client/object_put.go

```diff
@@ -19,10 +19,15 @@ import (
 )
 
 // PrmObjectPutInit groups parameters of ObjectPutInit operation.
-//
-// At the moment the operation is not parameterized, however,
-// the structure is still declared for backward compatibility.
-type PrmObjectPutInit struct{}
+type PrmObjectPutInit struct {
+	copyNum uint32
+}
+
+// SetCopiesNumber sets number of object copies that is enough to consider put successful.
+// Zero means using default behavior.
```

Comment:
could we specify what default behavior is? OR do not mention *node's* behavior in the client at all (which is the proper way IMO)?